vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,161 @@
----
-sidebarDepth: 2
----
 # Role Based Access Controls
 
 ## Overview
-This tutorial guides you through adding access controls to your existing Entando project. Security experts recommend following a practice known as `Defense in Depth` where security controls are placed in each layer of an architecture. This tutorial will help you setup such controls in both the frontend and backend of your Entando application. 
+Experts recommend following a practice known as Defense in Depth, where security controls are placed in each layer of an architecture. This tutorial guides you through adding access controls to your existing Entando project, in both the frontend and backend of your Entando Application. 
```

Comment:
Experts recommend following a practice known as Defense in Depth where security controls are placed in each layer of an architecture. >> no comma, I think the second part is essential to the first part as it is an explanation of the first.

Reply:
ignore the natural pause?